source/best_practices.rst

```diff
@@ -124,3 +124,84 @@ To save time, like ``zstash extract``, you can check for specific files or tar a
 
   $ zstash check --hpss=/path/to/hpss/archive "archive/ocn/hist/mpaso.hist.am.timeSeriesStatsMonthly.1892-04-01.nc"
   $ zstash check --hpss=/path/to/hpss/archive "000012.tar"
+
+Compy/Anvil
+===========
+
+There is no long-term HPSS storage attached to Compy or Anvil. To archive a new simulation, 
+we recommend the following:
+
+* Use zstash to create a local archive on disk.
+* Transfer files to NERSC HPSS using Globus.
+
+While the instructions below are specific for Compy, adapting them for Anvil should
+be straightforward.
+
+Archive
+-------
+
+Starting with v0.4, zstash supports the creation of local archives only (using the 
+``--hpss=none`` command line option). For example ::
+
+   $ screen
+   $ cd /compyfs/gola749/E3SM_simulations/20191216.alpha20.piControl.ne30_r05_oECv3_ICG.compy
+   $ mkdir zstash
+   $ zstash create --hpss=none  --maxsize 128 . 2>&1 | tee zstash/zstash_create_20200224.log
+   ctrl-a d # to disconnect from screen session
+
+Check
+-----
+
+Once archiving is complete, run ``zstash check`` locally to verify integrity of archive: ::
+
+   $ screen -r
+   $ cd /compyfs/gola749/E3SM_simulations/20191216.alpha20.piControl.ne30_r05_oECv3_ICG.compy
+   $ zstash check . 2>&1 | tee zstash/zstash_check_20200225.log
+
+Transfer to NERSC HPSS
+----------------------
+
+After the check completes successfully, transfer all zstash files to NERSC HPSS using 
+Globus.
+
+* Login to Globus web interface at https://www.globus.org/ using your NERSC credentials.
+* On the leftmost pane, select 'ENDPOINT'
+* Search for 'NERSC HPSS'. Click on Green power button to activate endpoint.
+
+.. image:: globus/Globus_Screenshot_1.png
```

Comment:
I'm not overly familiar with fair use / copyright law; since we didn't develop Globus, do we need to get legal permission to use screenshots from it?

Reply:
I will assume this falls under fair use unless we are told otherwise.